### ISSUE TITLE
Internal voltage domain bypass caps

### ISSUE THREAD

**Issue by kingovchouffe** [skot/bitaxe#197]
Hello,

I'm a hardware design engineer, and the part in the schematic of the bypass caps for the internals voltages domains makes absolutely no sense electrically speaking. 

Do you have documentation about the miner chip that specified this arrangement? 

I am willing to help with the hardware design, this project looks very cool !

**Comment by skot** [comment]
Hey, welcome!

There is no documentation... All we have are some cryptic "repair guides" and closely looking at existing hashboards.

**Comment by kingovchouffe** [comment]
I tried to look at some documentation about those ASIC but there is not much appart those repair guides which you mentioned. My theory is that their retro engineering is wrong in the symbol pin out, but someone it works. But bypassing two DC voltages with capacitors makes nonsense to me. I'll try to find some better picture of Asics board

**Comment by tkircher** [comment]
According to this, there should be no connections to VDD1, VDD2, or VDD3 in a single chip configuration:

https://www.zeusbtc.com/articles/information/4915-antminer-s19-xp-hydro-hash-board-repair-guide

**Comment by skot** [comment]
Are you referring to this diagram? ![image](https://github.com/user-attachments/assets/c93b35c4-a590-42bc-8e14-fa3f94ce6793)

That appears to be a higher level diagram. If you look at the actual hashboard you'll see each internal voltage domain has bypass capacitors.

**Comment by tkircher** [comment]
You're right, I didn't recognize that at first. I found this for the BM1397 that has the same basic design:
<img width="570" alt="BM1397_sch" src="https://github.com/user-attachments/assets/18345fb7-7cd1-463a-b18c-3e3538aeabc2">
And this:
![1714683399551](https://github.com/user-attachments/assets/221e92d5-6531-4e03-8bde-e611fedc2e74)
@kingovchouffe is right that this is a pretty unusual arrangement.
